### DISCUSSION TITLE
Start sensor initiated from the stop sensor class!

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#15] [Development]
The following image shows a section of the current (Jan. 12, 2021) StopSensor.java file.

![Screenshot 2021-05-15 084435](https://user-images.githubusercontent.com/51497406/118361639-5bfe2b00-b55a-11eb-94b2-ac3d092aa47d.png)

It seems that if the sensor is not active, the sensor will be started!  Why?  
If the user is on the stop sensor page, why would we ever start a sensor?

**Comment by tolot27** [comment]
I recommend to use the source code linking (of several lines) instead of creating images. Linked source code is shown in place.
